### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by matty45** [Sinytra/Connector#38] [General]
This is not a question or anything but more of an appreciation of the work that you have been doing here!

Was not sure if there was any effort made to at least bridge the gap between mod-loaders so we could have the best of both worlds.

I knew patchwork was a thing but as far as i know, its been abandoned.

Been playing Minecraft since 2011 and seeing some of my old favorites still being locked to one mod-loader was saddening to me 
(I use fabric mainly.), of course i could try porting them myself but i lack the knowledge and the motivation to do so.

It is so nice to see that a new effort has been made to make fabric mods work on forge or vice versa.

Have high hopes to see this project succeed and not to collapse or fall into development hell like the other ones.

**Comment by Cixon** [comment]
agreed. we love Su5eD !
